content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Announcing LocalStack 3.0 General Availability!
+description: We are excited to announce the release of LocalStack 3.0 featuring enhanced AWS services, Chaos Engineering tools, IAM Policy Stream, Desktop Application, improved performance and more!
+lead: We are excited to announce the release of LocalStack 3.0 featuring enhanced AWS services, Chaos Engineering tools, IAM Policy Stream, Desktop Application, improved performance and more!
+date: 2023-11-13
+lastmod: 2023-11-13
+images: ['localstack-3.0.png']
+contributors: ["LocalStack Team"]
+tags: ['news']
+show_cta_1: true
+leadimage: 'localstack-3.0.png'
+---
+
+{{< img-simple src="localstack-3.0.png" >}}
+
+Today, we are excited to announce **LocalStack 3.0**, our latest major release towards empowering developers with a high-fidelity local cloud development & testing experience. Earlier this year, we unveiled [LocalStack 2.0](https://localstack.cloud/blog/2023-03-29-announcing-localstack-2.0-general-availability/), which brought forth a host of new features and improvements, along with better integration with the AWS tooling ecosystem. With the 3.0 release, we're doubling down on our promise to boosting developer productivity and tackling the issues that often slow down cloud software development and delivery workflows — specifically, simplifying onboarding and eliminating the inefficiencies that lead to prolonged dev&test cycles. Additionally, this release represents a significant milestone in our journey as we move beyond traditional cloud emulation, and introduce innovative features that promise to reshape the way developers construct resilient cloud applications.
```

Comment:
'dev&test' -> 'development and test' or at least 'dev and test'

Reply:
We have used `dev&test` exclusively now. I understand that it's grammatically not super correct 😅 

Maybe @whummer can chime in?

Reply:
Either way is fine for me - we've used `dev&test` as a shorthand, but also fine to write it out in full.. 👍